initial-setup-gui.sh

```diff
@@ -22,6 +22,26 @@ TESTTYPE="initial-setup"
 . ${KSTESTDIR}/functions.sh
 . ${KSTESTDIR}/validate-lib-initial-setup.sh
 
+prepare() {
+    ks=$1
+    tmpdir=$2
+    export_additional_repo $tmpdir
+    echo $ks
+}
+
+kernel_args() {
+    echo $(append_additional_repo_to_kernel_args $DEFAULT_BOOTOPTS)
+}
+
+cleanup() {
+    tmpdir=$1
+
+    # cleanup the localhost webserver we might have started
+    if [ -f ${tmpdir}/httpd-pid ]; then
```

Comment:
You can call the `stop_httpd` function.

Reply:
Actually, this I left the changes in initial-setup-gui.sh by mistake, I'll drop it from there and only leave it in the Initial Setup reboot tests as we discussed earlier.